tests/frontend/test_versions.py

```diff
@@ -0,0 +1,139 @@
+                    )
+                )
+
+            else:  # if link exists and is not 'Custom License'
+                expected_link = page.versions_list[i].license_link.get_attribute('href')
+                if 'http' or 'https' in expected_link:
+                    # full url example-> https://example.com/something/something
+                    # turn into-> example.com/something/something
+                    expected_link = expected_link.split('//')[1]
+                    # turn into-> example.com
+                expected_link = expected_link.split('/')[0]
+                page.versions_list[i].license_link.click()
+                assert expected_link in selenium.current_url
+
+            # in both cases, we go back
+            page.driver.back()
+            page.driver.refresh()
+        else:  # if link does not exist
+            assert 'All Rights Reserved' in page.versions_list[i].license_text
+
+
+@pytest.mark.nondestructive
+def test_current_version(selenium, base_url):
+    addon_url = 'addon/devhub-listed-ext-07-30/'
+    # get info from api
+    response = requests.get(f'{base_url}/api/v5/addons/{addon_url}')
+    addon_version = response.json()['current_version']['version']
+    addon_size_kb = (
+        str(round(response.json()['current_version']['file']['size'] / 1024, 2)) + ' KB'
+    )
+    api_date = response.json()['current_version']['file']['created'][:10]
+    api_creation_date = datetime.strptime(api_date, '%Y-%m-%d')
+    # verify info displayed in page
+    page = Versions(selenium, base_url)
+    selenium.get(f'{base_url}{ADDON_VERSION_PAGE}')
+    assert page.versions_list[0].version_number == addon_version
+    assert addon_size_kb == page.versions_list[0].version_size
+    frontend_date = page.versions_list[0].released_date
+    # Special case: if date is 'Dec 9, 2021' it should be 'Dec 09, 2021', so we must add that '0'
+    if frontend_date[5] is ',':
+        frontend_date = frontend_date[:4] + '0' + frontend_date[4:]
+    frontend_creation_date = datetime.strptime(frontend_date, '%b %d, %Y')
+    assert frontend_creation_date == api_creation_date
+
+
+@pytest.mark.nondestructive
+def test_version_install_warning(selenium, base_url, variables):
+    selenium.get(
+        f'{base_url}/en-US/firefox/addon/{variables["non_recommended_addon"]}/versions/'
+    )
+    page = Versions(selenium, base_url)
+    for version in page.versions_list:
+        assert variables["install_warning_message"] in version.warning_message.text
+        version.warning_learn_more_button.click()
+        page.driver.switch_to.window(page.driver.window_handles[1])
+        page.wait_for_title_update('Add-on Badges')
+        page.driver.switch_to.window(page.driver.window_handles[0])
+
+
+@pytest.mark.nondestructive
+def test_add_to_firefox_button(selenium, base_url, firefox, firefox_notifications):
+    selenium.get(f'{base_url}/en-US/firefox/addon/donkey-kong/versions/')
```

Comment:
do not hardcode the add-on in the test; use a variable.

Reply:
Ok